atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -336,6 +336,7 @@ private static TransactionManager createInMemory(
                 .globalMetricsRegistry(new MetricRegistry())
                 .globalTaggedMetricRegistry(DefaultTaggedMetricRegistry.getDefault())
                 .lockAndTimestampServiceFactory(maybeFactory)
+                .defaultTimelockClientFactory((service, _management) -> TimeLockClient.withSynchronousUnlocker(service))
```

Comment:
This makes is so we do not get batching for some endpoints (freshTs):

```
        RequestBatchingTimestampService timestampService =
                RequestBatchingTimestampService.create(new TimelockServiceErrorDecorator(timelockService));
```

We probably still want this? Saves up on cycles / threads

Reply:
I would be surprised if it matters in most tests, where you just have a single thread, period. But maybe for internal skiing product integration tests? Lemme have a quick look

Reply:
I am though going to write a quick search internally to usages of TransactionManagers#createInMemory, just to make sure this is used the way I think it is